.circleci/config.yml

```diff
@@ -18,6 +18,26 @@ anchor_test_ruby: &anchor_test_ruby
     bundle exec rake test
   when: always
 
+anchor_test_php: &anchor_test_php
+  command: |
+    # Only run PHP API test if we have project credentials key
+    # or this API client doesn't require credentials.
+    if ! [ -z "$SYSTEM_TESTS_ENCRYPTION_KEY" ] || [ "$KEY_REQUIRED" = false ]
+    then
+      cd workspace
+      git clone \
+          https://github.com/GoogleCloudPlatform/google-cloud-php-${TEST_API}.git
+      cd google-cloud-php-${TEST_API}
+      cp -r ${PHP_CLIENTS_DIR}/google-cloud-pubsub-*/src \
```

Comment:
`google-cloud-pubsub-*`? Should this be `google-cloud-php-*`? Or `google-cloud-${TEST_API}-*`?